ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -385,6 +385,37 @@ if test "${ret}" -eq 0 ; then
   touch  "${SHARED_DIR}/success"
   # Save console URL in `console.url` file so that ci-chat-bot could report success
   echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
+
+  if [ ! -f /tmp/jq ]; then
+    curl -L https://stedolan.github.io/jq/download/linux64/jq -o /tmp/jq && chmod +x /tmp/jq
+  fi
+  #save cluster data
+  touch  "${ARTIFACT_DIR}/cluster-data"
+  # platform
+  echo "Platform: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get infrastructure/cluster -o json|/tmp/jq '.status.platform')/n" >> "${ARTIFACT_DIR}/cluster-data"
+  # topology
+  TOPOLOGY="$(env KUBECONFIG=${dir}/auth/kubeconfig oc get node -o json|/tmp/jq '.items[0]|.metadata.annotations'|grep machineconfiguration.openshift.io/controlPlaneTopology)"
+  if [[ "$TOPOLOGY" =~ .*"HighlyAvailable".*  ]]; then
+    echo "Topology: ha/n" >> "${ARTIFACT_DIR}/cluster-data"
+  elif [[ "$TOPOLOGY" =~ .*"Single".*  ]]; then
+    echo "Topology: single/n" >> "${ARTIFACT_DIR}/cluster-data"
+  else
+    echo "Topology: external/n" >> "${ARTIFACT_DIR}/cluster-data"
+  fi
+  # network type
+  echo "Network Type: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get network.operator cluster -o json|/tmp/jq '.spec.defaultNetwork.type')/n" >> "${ARTIFACT_DIR}/cluster-data"
+  # network stack
+  if [[ "$(env KUBECONFIG=${dir}/auth/kubeconfig oc get network.operator cluster -o json|/tmp/jq '.spec.clusterNetwork[0].cidr')" =~ .*":".*  ]]; then
+    echo "Network Stack: IPv6/n" >> "${ARTIFACT_DIR}/cluster-data"
+  else
+    echo "Network Stack: IPv4/n" >> "${ARTIFACT_DIR}/cluster-data"
+  fi
+  # cluster version history
+  echo "Cluster Version History: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get clusterversion -o json|/tmp/jq '.items[]|.status.history'|grep version|cut -d : -f 2| sort -u)/n" >> "${ARTIFACT_DIR}/cluster-data"
+  # cloud region
+  echo "Cloud Region: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get node -o json|/tmp/jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/region|cut -d : -f 2| sort -u)/n" >> "${ARTIFACT_DIR}/cluster-data"
+  # cloud zone
+  echo "Cloud Zone: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get node -o json|/tmp/jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/zone|cut -d : -f 2| sort -u)" >> "${ARTIFACT_DIR}/cluster-data"
```

Comment:
What will we get here if this doesn't exist? I assume on some platforms it won't like metal?

Reply:
What will Cloud Zone be though?